src/Authentication/JWTAuthenticationHandler.php

```diff
@@ -70,14 +68,7 @@ public function logIn(Member $member, $persistent = false, HTTPRequest $request
      */
     public function logOut(HTTPRequest $request = null): void
     {
-        // A token can actually not be invalidated, but let's flush all valid tokens from the DB.
-        // Note that log-out acts as a global logout (all devices)
-        /** @var Member|MemberExtension $member */
-        $member = Security::getCurrentUser();
-        if ($member) {
-            $member->destroyAuthTokens();
-        }
-
-        Security::setCurrentUser(null);
+        // We don't take any action here.
```

Comment:
Then the token stays valid, which is even worse than logging out from everywhere...